packages/app-store/zoomvideo/lib/VideoApiAdapter.ts

```diff
@@ -196,11 +196,12 @@ const ZoomVideoApiAdapter = (credential: CredentialPayload): VideoApiAdapter =>
     };
 
     const recurrence = getRecurrence(event);
+    const utcOffset = dayjs(event.startTime, event.organizer.timeZone).utcOffset() / 60;
```

Comment:
@zomars @joeauyeung I believe this code `dayjs(event.startTime, event.organizer.timeZone)` is resulting in `NaN`, which is setting the `start_time` on L204 to be the current time.

I can reproduce this consistently:

https://github.com/calcom/cal.com/issues/6928#issuecomment-1470975620

I just tried it again today at 2:47pm with the following parameters:

Scheduled the meeting for 3/18 at 11am.

<img width="670" alt="image" src="https://user-images.githubusercontent.com/7423098/225736785-21fdb863-b04a-4b8b-adee-1403b9be6b8e.png">

And the zoom meeting was scheduled for today at 2:47pm.

<img width="631" alt="image" src="https://user-images.githubusercontent.com/7423098/225736889-4afd58ca-e2e8-4935-bee7-986f8751a96e.png">

Reply:
Thanks for finding this @lottamus. I'll look into it